tests/test_species.py

```diff
@@ -151,6 +151,15 @@ def test_generation_time_defined(self):
     def test_population_size_defined(self):
         assert self.species.population_size > 0
 
+    def test_default_gc(self):
+        for chrom in self.species.genome.chromosomes:
+            contig = self.species.get_contig(chrom.id)
+            assert chrom.gene_conversion_rate is None
```

Comment:
I think this line should be 
`assert contig.gene_conversion_rate is None`
and
`assert contig.gene_conversion_length is None`
and this should fix the test error.

Reply:
hah, good catch!